antlr-kotlin-target/src/main/kotlin/org/antlr/v4/codegen/target/KotlinTarget.kt

```diff
@@ -21,20 +22,22 @@ public class KotlinTarget(codeGenerator: CodeGenerator) : JavaTarget(codeGenerat
       "if",
       "implements",
       "import",
-      "is",
       "interface",
+      "internal",
```

Comment:
This seems to contain a mix of hard keywords, soft keywords, and modifier keywors (see [Kotlin's keyords](https://kotlinlang.org/docs/keyword-reference.html)? )

I would make revise this list based on the documentation. For example, `as` and `break` are missing

Reply:
Great, thank you!